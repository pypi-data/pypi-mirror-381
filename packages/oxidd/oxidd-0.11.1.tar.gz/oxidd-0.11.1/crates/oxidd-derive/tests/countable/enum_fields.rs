use oxidd_derive::Countable;

#[derive(Countable)]
enum Tag {
    X(u8),
    Y(u32),
}

fn main() {}
