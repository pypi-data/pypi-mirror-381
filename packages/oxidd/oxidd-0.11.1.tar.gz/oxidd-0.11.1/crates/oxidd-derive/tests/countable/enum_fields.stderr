error: can only derive `Countable` for `enum`s with `repr(u8)`
 --> tests/countable/enum_fields.rs:4:1
  |
4 | / enum Tag {
5 | |     X(u8),
6 | |     Y(u32),
7 | | }
  | |_^

error: can only derive `Countable` for fieldless `enum`s
 --> tests/countable/enum_fields.rs:5:6
  |
5 |     X(u8),
  |      ^^^^

error: can only derive `Countable` for fieldless `enum`s
 --> tests/countable/enum_fields.rs:6:6
  |
6 |     Y(u32),
  |      ^^^^^
